.eslintrc.js

```diff
@@ -48,16 +48,6 @@ module.exports = {
     // `function foo(...args: readonly unknown[])`)
     'prefer-rest-params': 'off',
 
-    // TODO: These all have autofixers
-    '@typescript-eslint/array-type': 'off',
```

Comment:
Can we set this to prefer `Array<>` over `[]` ?

```
'@typescript-eslint/array-type': ['error', { default: 'generic', readonly: 'generic' }],
```

I'm aware it's a preferential choice 😅 but there are a couple of reasons why I think this syntax is "objectively better"

Reply:
thanks! but why level `warn` and not `error` ?

Reply:
I had a sense that the winds are shifting towards "array" styling, to me it just makes it more obvious to non-native Typescript readers that this is a simple "native" array and not something special... (also you can't redefine the Array type and completely mess with people's head by accident)

Reply:
> thanks! but why level `warn` and not `error` ?

It's the default for this rule.

Reply:
but we're not enforcing zero warnings, right? In my experience, eslint warnings are often just ignored

Reply:
I never put much thought to why something would be warning and something else error. I personally would still comment in a PR which has a lint warning, and they are both suppressesable all the same using a lint disable comment.

My only thought of keeping it a warning is that it communicates better to anyone who'll reconsider this rule in the future that the original author didn't think this rule is critical so it might be ok to change it (versus a rule set as error), but that's pretty much it, i dunno...

I don't mind changing it, I actually didn't notice your code snippet in the original comment, I just saw you said you like generics and went ahead and changed it...

Reply:
thanks 🙏